next.config.js

```diff
@@ -50,6 +50,10 @@ const { withSentryConfig } = require("@sentry/nextjs");
 module.exports = withSentryConfig(
   module.exports,
   {
+    // For all available options, see:
```

Comment:
Seems to be an older commit you are working off because these commits aren't needed.